docs/configurations.md

```diff
@@ -165,59 +196,82 @@ The following configurations are available for JDBC transactions:
 
-In addition to [transaction manager configurations](#transaction-manager-configurations) and [other configurations](#other-configurations), the following configurations are available for ScalarDB Server:
+In addition to [transaction managers](#transaction-managers) and [other configurations](#other-configurations), the following configurations are available for ScalarDB Server:
 
-| Name                                              | Description                                                                                      | Default                |
-|---------------------------------------------------|--------------------------------------------------------------------------------------------------|------------------------|
-| `scalar.db.server.port`                           | Port number for ScalarDB Server.                                                                 | `60051`                |
-| `scalar.db.server.prometheus_exporter_port`       | Prometheus exporter port. Prometheus exporter will not be started if a negative number is given. | `8080`                 |
-| `scalar.db.server.grpc.max_inbound_message_size`  | The maximum message size allowed to be received.                                                 | The gRPC default value |
-| `scalar.db.server.grpc.max_inbound_metadata_size` | The maximum size of metadata allowed to be received.                                             | The gRPC default value |
-| `scalar.db.server.decommissioning_duration_secs`  | The decommissioning duration in seconds.                                                         | `30`                   |
+| Name                                              | Description                                                                                      | Default                 |
+|---------------------------------------------------|--------------------------------------------------------------------------------------------------|-------------------------|
+| `scalar.db.server.port`                           | Port number for ScalarDB Server.                                                                 | `60051`                 |
+| `scalar.db.server.prometheus_exporter_port`       | Prometheus exporter port. Prometheus exporter will not be started if a negative number is given. | `8080`                  |
+| `scalar.db.server.grpc.max_inbound_message_size`  | The maximum message size allowed to be received.                                                 | The gRPC default value. |
+| `scalar.db.server.grpc.max_inbound_metadata_size` | The maximum size of metadata allowed to be received.                                             | The gRPC default value. |
+| `scalar.db.server.decommissioning_duration_secs`  | The decommissioning duration in seconds.                                                         | `30`                    |
 
 For details about ScalarDB Server, see [ScalarDB Server](scalardb-server.md).
 
```

Comment:
(Same question about ScalarDB Server as in my [earlier comment](https://github.com/scalar-labs/scalardb/pull/1082/files#r1322586289).)

I don't think this section should be in the docs in the `master`, `3.10`, and `3.9` branches since ScalarDB Server has been deprecated as of ScalarDB 3.9.

For ScalarDB docs on the `master`, `3.10`, and `3.9` branches, should we remove this section or point users to [ScalarDB Cluster Configurations](https://github.com/scalar-labs/scalardb-cluster/blob/main/docs/scalardb-cluster-configurations.md)?